.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+jdk:
+  - openjdk8
```

Comment:
We should open an issue for deciding if we are going to support/test other versions of java (unrelated with the acceptance of this build, but just came in this pass)

Reply:
Opened #18